docs/content/installation/install_compatibility_requirements.md

```diff
@@ -95,10 +95,16 @@ Complete these steps for each worker node in Kubernetes cluster to prepare your
     
     2. To enable support on your storage system, see the following section within your Spectrum Virtualize product documentation on [IBM Documentation](https://www.ibm.com/docs/en/): **Administering** > **Managing Copy Services** > **Managing remote-copy partnerships**.
 
-6. (Optional) To use CSI Topology, at least one node in the cluster must have the label-prefix of `topology.block.csi.ibm.com` to introduce topology awareness:
+6. (Optional) To use CSI Topology, at least one node in the cluster must have the label-prefix of `topology.block.csi.ibm.com` to introduce topology awareness.
```

Comment:
we might want to add a reference for more information about CSI topology and what it is, not just how to use it and configure it. there's an appropriate reference [here](https://www.ibm.com/docs/en/stg-block-csi-driver/1.7.0?topic=notes-whats-new-in-170)
I'm not sure if this is the right location to where we should add it though

Reply:
the question is - what do the users who we are expecting to be using the documentation know or not about the topic and usage. if you think that the general csi user might need more info about csi topology, please open a csi doc ticket with the information so i can add it in in a future release. thanks

Reply:
ok, thanks.